docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,810 @@ thor:
+
+            <programlisting>kubectl get pods -n hpcc2</programlisting>
+
+            <para>Check and see if the certificate issuers have been
+            successfully created:</para>
+
+            <programlisting>kubectl get issuers -o wide</programlisting>
+          </sect5>
+
+          <sect5 id="ECLExampleDemonstratingTrust">
+            <title>ECL Example Demonstrating Trust</title>
+
+            <para>Now publish ECL to each environment that will communicate
+            securely across the environments.</para>
+
+            <para>roxie_echo.ecl which returns a dataset passed into it.
+            remote_echo.ecl which calls roxie_echo.ecl.</para>
+
+            <para>For this example we will:</para>
+
+            <orderedlist>
+              <listitem>
+                <para>publish roxie_echo.ecl to the hpcc1 namespace. It
+                returns the dataset passed into it.</para>
+              </listitem>
+
+              <listitem>
+                <para>Publish remote_echo.ecl to the hpcc2 namespace. It calls
+                roxie_echo.ecl in the hpcc1 namespace using remote mTLS. This
+                is accomplished by using the <literal>remote-mtls:</literal>
+                prefix in the url to signal that the call should be made using
+                the certificate associated with the <literal>remote</literal>
+                issuer.</para>
+              </listitem>
+
+              <listitem>
+                <para>Use hpcc2::remote_echo.ecl to call
+                hpcc1::roxie_echo.ecl.</para>
+              </listitem>
+            </orderedlist>
+
+            <para>Publish the queries:</para>
+
+            <programlisting>ecl publish roxie1 --ssl --port 18010 roxie_echo.ecl
+ecl publish roxie2 --ssl --port 28010 remote_echo.ecl</programlisting>
+
+            <para>Call the query and demonstrate trust</para>
+
+            <para>You can navigate to EclQueries/WsEcl on port 28002 in your
+            browser and run the <literal>remote_echo</literal> query from
+            there. You should see results similar to the following:</para>
+
+            <programlisting language="json">{"remote_echoResponse":  {"sequence": 0, "Results":  {"remoteResult": {"Row": [{"Dataset": {"Row": [{"name": {"first": "Joeseph", "last": "Johnson"}, "address": {"city": "Fresno", "state": "CA", "zipcode": "11111"}}, {"name": {"first": "Joeseph", "last": "Johnson"}, "address": {"city": "Fresno", "state": "CA", "zipcode": "22222"}}, {"name": {"first": "Joeseph", "last": "Johnson"}, "address": {"city": "Fresno", "state": "CA", "zipcode": "33333"}}, {"name": {"first": "Joeseph", "last": "Johnson"}, "address": {"city": "Fresno", "state": "CA", "zipcode": "44444"}}, {"name": {"first": "Joeseph", "last": "Johnson"}, "address": {"city": "Fresno", "state": "CA", "zipcode": "55555"}}]}, "Exception": {"Code": "0"}}]}}}}}</programlisting>
+          </sect5>
+        </sect4>
+
+        <sect4 id="SSLTLSTroubleshooting">
```

Comment:
In this "SSLTLSTroubleshooting" section, it makes sense to remove SSL. Also I think it would be better placed after the the next sect4 "UsingPublicCA"